posts/share-cpp/index.qmd

```diff
@@ -96,14 +96,14 @@ We would however recommend not autogenerating headers from Rcpp functions, but r
 There are few to no documented rules around the content of the package header.
```

Comment:
In line 62 above, wasn't clear to me if that command generates a header file under `inst/include` AND a header file named `mypackage_RcppExports.h`, or if these are the same thing?

Reply:
Hopefully clearer after https://github.com/epiverse-trace/epiverse-trace.github.io/pull/34/commits/8efbb4ed0c36569855d9154a757e99a98af3d77c?